### ISSUE TITLE
Set datum for `coops_product_within_region()`

### ISSUE THREAD

**Issue by FariborzDaneshvar-NOAA** [oceanmodeling/StormEvents#89]
I noticed that default value  for `datum` in the [coops_product_within_region()](https://github.com/oceanmodeling/StormEvents/blob/79f5ab1d145600dd758179af1d314cf580b935eb/stormevents/stormevent.py#L426) function in stormevent.py, is `None` which will return `STND` datum (see example plot below). 

![image](https://github.com/oceanmodeling/StormEvents/assets/132295102/419f7a21-faa4-4355-9d34-067678e61823)

Users can set `datum=NAVD` or any other value from the following list depending to their use
`datum_list=['STND', 'MHHW', 'MHW', 'MTL', 'MSL', 'MLW', 'MLLW', 'NAVD']`

I would suggest changing the default to `NAVD` or `MSL` as these two are more commonly used than `STND`, and/or adding the list of possible options to line [446 ](https://github.com/oceanmodeling/StormEvents/blob/79f5ab1d145600dd758179af1d314cf580b935eb/stormevents/stormevent.py#L446).

@SorooshMani-NOAA, Let me know your thoughts. If you don't mind, I can make these changes.

**Comment by SorooshMani-NOAA** [comment]
@FariborzDaneshvar-NOAA I agree that MSL or NAVD88 makes more sense to have. Note that we're also in the middle of updating the searvey's data API for COOPS. Feel free to update the default and start a PR, but please keep this open since I'm already working on the new API.

**Comment by SorooshMani-NOAA** [comment]
@FariborzDaneshvar-NOAA in the COOPS API documentation it says

> NAVD: North American Vertical Datum - Note! This datum is not available for all stations.

**Comment by FariborzDaneshvar-NOAA** [comment]
@SorooshMani-NOAA Thanks! So how about using `MSL` as default?

**Comment by SorooshMani-NOAA** [comment]
I think it would be safer, let's go with MSL for now

**Comment by FariborzDaneshvar-NOAA** [comment]
https://github.com/oceanmodeling/StormEvents/pull/90
